app/assets/javascripts/templates/analysis-total.hbs

```diff
@@ -1,18 +1,28 @@
-<div class="text -analysis-total">
-  {{#if totalCompare}}
-    Introducing alternative practices would result in:
-  {{else}}
-    Your current estimated emission are:
-  {{/if}}
-</div>
-<div class="total">
+{{#if totalCompare}}
+  <div class="main-figures">
+    <div>
+      <span class="title text -summary-small">Current practices</span>
+      <p class="value text -summary-big">{{total}}</p>
+    </div>
+    <div>
+      <span class="title text -summary-small">Potential future</span>
+      <p class="value text -summary-big">{{totalCompare}}</p>
+    </div>
+  </div>
+  <p class="text -analysis-total -bold">Introducing alternative practices would result in:</p>
+  <div class="susbtraction">
+    <p class="value text -figure-extra-big -highlighted">{{substraction}}</p>
```

Comment:
I'd suggest making the number green if it's negative and red if it's positive to show if the impact is positive or negative.